src/msg.rs

```diff
@@ -54,6 +58,11 @@ pub enum ExecuteMsg {
         min_token: Uint128,
         expiration: Option<Expiration>,
     },
+    UpdateFees {
```

Comment:
Wonder if we should call this `UpdateConfig` and allow updating the owner as well?

Reply:
Done